apps/account-ui/src/app.module.css

```
@@ -0,0 +1,4 @@
+
+img.logo {
```

Let's not make this selector overly specific.
```suggestion
.logo {
```

---

it's needed otherwise it doesn't override patternfly